config/src/config/consensus_config.rs

```diff
@@ -59,6 +72,37 @@ impl Default for ConsensusConfig {
             quorum_store_pull_timeout_ms: 1000,
             quorum_store_poll_count: 10,
             intra_consensus_channel_buffer_size: 10,
+
+            window_for_chain_health: 100,
+            chain_health_backoff: vec![
+                ChainHealthBackoffValues {
+                    backoff_if_below_participating_voting_power_percentage: 80,
+                    max_sending_block_txns_override: 2000,
```

Comment:
What's the relationship between txns and bytes? Why is it not linear?

Reply:
I rounded it up, it should be pretty linear